index.js

```diff
@@ -5,46 +5,48 @@ var through = require('through2');
 var minimatch = require("minimatch");
 var _ = require('lodash');
 
-function instrument(options) {
-  var Instrumenter = require(resolve.sync(options.instrumenter, {basedir: __dirname}));
+function filterFiles(options, files) {
   var excludePattern = options.exclude ? [].concat(options.exclude) : [''];
```

Comment:
I was trying hard to not break the API. It would be great to release this as a feature. The files in the `node_modules` folder are not passed to transforms by default. Only global transforms receive all files.